docs/ko/translation-guide.md

```diff
@@ -267,32 +267,32 @@ A value of 0.01 was used for the value to ramp down to in the last function rath
 | 용어 | 번역 | 참고 링크 |
 | --- | --- | --- |
 | Accessibility concerns | 접근성 고려사항 | |
-| Advantages | 장점 | [링크](https://developer.mozilla.org/en-US/docs/Games/Introduction_to_HTML5_Game_Development) |
-| Aliasing | 별칭 | [링크](https://github.com/mdn/translated-content/pull/1779/files) |
-| Brief history | 간략한 역사 | [링크](https://developer.mozilla.org/ko/docs/Glossary/HTML#%EA%B0%84%EB%9E%B5%ED%95%9C_%EC%97%AD%EC%82%AC) |
-| Browser compatibility | 브라우저 호환성 | [링크](https://github.com/mdn/translated-content/pull/1779/files) |
-| Browser support | 브라우저 지원 | [링크](http://localhost:5042/en-us/docs/Web/API/Web_Audio_API/Using_IIR_filters#browser_support) |
+| Advantages | 장점 | <https://developer.mozilla.org/en-US/docs/Games/Introduction_to_HTML5_Game_Development> |
+| Aliasing | 별칭 | <https://github.com/mdn/translated-content/pull/1779/files> |
```

Comment:
Why do we use no link text here? Is it related to the rendering issues you mentioned in the description? Looks like the same link text `링크` (link?) is used in all of these.

Reply:
Yes, I couldn't move to a reference link since they're the same text. I guess I could have used non-matching reference values, but the "Link" as text is also usually an accessibility issue so I just unfolded them

Reply:
The reason why the link text exists is that it reduces readability if the URL is entered when viewed on a mobile device.

![image](https://user-images.githubusercontent.com/22424891/211623686-0502fc1f-c6cb-418d-9103-998cf1901937.png)
![image](https://user-images.githubusercontent.com/22424891/211623799-70fce324-ee19-492f-8e79-31b85885bbf7.png)

I would like to keep the current direction, but is there any way?

Reply:
The only way is to assign each one a unique name. (e.g. link_1)

Reply:
This looks like a "Advantages Example Link" like "장점 예시 링크", so if you solve the conflict, I'll suggest it as a review.

Reply:
I found a way to just use reference links. It's still a little odd, but keeps the format you have now